weave

```diff
@@ -1211,6 +1212,7 @@ launch_proxy() {
         --entrypoint=/home/weave/weaveproxy \
         $WEAVEPROXY_DOCKER_ARGS $EXEC_IMAGE $COVERAGE_ARGS $PROXY_ARGS)
     wait_for_log $PROXY_CONTAINER_NAME "proxy listening"
+    ln -sf -T /var/run/weave/weave.sock /var/run/weave.sock
```

Comment:
This is there just for backward compatibility, right? Please a) add a comment to say so, and b) file an issue to remove this and the `/var/run` volume mount in `exec_remote` for 1.2.